docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md

```diff
@@ -29,14 +29,17 @@ Here is a table of the currently supported networks on HyperRPC and their respec
 | Bsc Testnet          | 97         | https://bsc-testnet.rpc.hypersync.xyz or https://97.rpc.hypersync.xyz               |                 |
 | C1 Milkomeda         | 2001       | https://c1-milkomeda.rpc.hypersync.xyz or https://2001.rpc.hypersync.xyz            |                 |
 | Celo                 | 42220      | https://celo.rpc.hypersync.xyz or https://42220.rpc.hypersync.xyz                   |                 |
+| Chiliz               | 8888       | https://chiliz.rpc.hypersync.xyz or https://8888.rpc.hypersync.xyz                  |                 |
```

Comment:
Does this not need to be generated by the code?

Reply:
I didn't intend to change anything here on this page. might have made a space or something. it looks fine locally and vercel.